acceptance/Rakefile

```diff
@@ -9,9 +9,13 @@ ONE_DAY_IN_SECS = 24 * 60 * 60
 REPO_CONFIGS_DIR = "repo-configs"
 CLEAN.include('*.tar', REPO_CONFIGS_DIR, 'merged_options.rb')
 
+# If elsewhere we're depending on internal network resources
+# then assume we can depend on them here
+INTERNAL_NETWORK = ENV['GEM_SOURCE'] && ENV['GEM_SOURCE'] =~ /puppetlabs/
```

Comment:
Updated with `BEAKER_EPEL_MIRROR`

That follows the `BEAKER_*` env var scheme, which may be good or bad depending on whether you want to have all configuration for Beaker stored in `BEAKER_` or you want to differentiate env vars that are consumed by the rake task vs those consumed directly via Beaker (if so, perhaps a `PUPPET_*` pattern?).